coffee_core/src/coffee.rs

```diff
@@ -396,14 +395,17 @@ impl PluginManager for CoffeeManager {
         })
     }
 
-    async fn show(&mut self, plugin: &str) -> Result<Value, CoffeeError> {
+    async fn show(&mut self, plugin: &str) -> Result<CoffeeShow, CoffeeError> {
         self.remote_sync().await?;
         for repo in self.repos.values() {
             if let Some(plugin) = repo.get_plugin_by_name(plugin) {
                 // FIXME: there are more README file options?
                 let readme_path = format!("{}/README.md", plugin.root_path);
                 let contents = fs::read_to_string(readme_path).await?;
-                return Ok(json!({ "show": contents }));
+                let json_contents = json!({ "show": contents });
+                return Ok(CoffeeShow {
+                    readme: json_contents,
+                });
```

Comment:
this is a waste of time

Why should I have a JSON inside a struct that we use to avoid passing raw json around?
```suggestion
                return Ok(CoffeeShow {
                    readme: content,
                });
```